.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 /Cargo.lock
+cudnn.log
```

Comment:
Is this something cudnn creates?

Reply:
i have an env var to log all cudnn calls, i can remove that if necessary